build.gradle.kts

```diff
@@ -37,6 +37,11 @@ dependencies {
 
 	implementation("net.bytebuddy:byte-buddy:1.17.6")
 
+	compileOnly("org.projectlombok:lombok:1.18.38")
+	annotationProcessor("org.projectlombok:lombok:1.18.38")
+	testCompileOnly("org.projectlombok:lombok:1.18.38")
+	testAnnotationProcessor("org.projectlombok:lombok:1.18.38")
```

Comment:
I don't like lombok, to be real here. What's your opinion @4everTheOne ?

Reply:
Yep, that's why I'm not fully against it. Generally the boilerplate is a sign that your code is not ideally made, you can usually get around with normal java using records and such. For this case though, we can not avoid that, but there's also going to be less thought but into the method as it's just an annotation. The copy operation is very common to use in getters and setters in the api implementations, so might not do that much, and might be missleading.

Reply:
I get your point. The biggest downside i see with using it, that since we are replicating behavior of paper in unit testing, some of the setters require preconditions. If the preconditions are not in lombok, like an int range or it requires a custom message for example. These examples would require us to create the preconditions manually which means for that method we cannot use lombok. If this is the most common use case i feel like it may be unnecessary..

Reply:
I did not consider that, I don't think it's going to be that prevalent, and I also think most cases will not match, as it's a multi line regex, which does not match with the `/` or `*` symbol. Mostly me having fun though 😛

Reply:
@4everTheOne Your opinion, you rule over this matter now with the given data.